src/conditional-flow/numeric-conditions.js

```diff
@@ -4,13 +4,25 @@
 function isArrayEmpty (array) {
 
   // TODO: write code in this function body to pass the tests
+  if (array === []) {
```

Comment:
what you have coded will work; you can also choose to do the following:

`if(array.length === 0)` to test if the array is empty